TeknoParrotUi.Common/GameProfiles/GHA.xml

```diff
@@ -1,14 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <GameProfile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <GameName>Guitar Hero Arcade</GameName>
+  <GameName>Guitar Hero - Arcade</GameName>
```

Comment:
no - in official name https://rawthrills.com/games/games-archive/gha/

Reply:
OK, re-issued.